styles/simplified-styles.css

```diff
@@ -46,6 +46,12 @@ a:hover {
     color: var(--button-primary-border-hover);
 }
 
+li {
+    list-style-type: disc;
+    margin: 0 2.5rem;
```

Comment:
I think the right side margin is not required. 

![image](https://user-images.githubusercontent.com/1191451/232677973-2cdfe1af-4534-4e49-97e6-131d46b53ad0.png)

Reply:
Thanks @sdmcraft I fixed that now